deploy/docker-compose/docker-compose.yml

```diff
@@ -7,9 +7,9 @@ services:
     ports:
       - "50051:50051"
     volumes:
-      - ../../config:/app/config:ro
-      - ../../models:/app/models:ro
-      - ~/.cache/huggingface:/root/.cache/huggingface
+      - ../../config:/app/config:ro,z
```

Comment:
Why was the container permission changed from ro to z?
The key point here should be that the data being mounted should not be modified.